src/Ledger/Conway/Specification/Epoch.lagda.md

```diff
@@ -17,6 +17,7 @@ import      Data.Integer as ℤ
 open import Data.Integer.Properties using (module ≤-Reasoning; +-mono-≤; neg-mono-≤; +-identityˡ)
                                     renaming (nonNegative⁻¹ to nonNegative⁻¹ℤ)
 open import Data.Nat.GeneralisedArithmetic using (iterate)
+open import Data.Maybe using (fromMaybe)
```

Comment:
For my OCD, could you move this import above the `Data.Nat` import?  (`M` comes before `N`.)